package androidx.compose.material.icons.rounded;

import bj.n;
import g1.m0;
import g1.t;
import j0.c;
import k0.a;
import k0.b;
import k0.d;
import k1.e;
import k1.f;
import k1.h0;
import kotlin.jvm.internal.l;

/* JADX INFO: compiled from: r8-map-id-cf66f36031e302b9a7b5a76da57d208ae1b6ba4a38bba3bee1d4ed6983c54df3 */
/* JADX INFO: loaded from: classes.dex */
public final class BorderStyleKt {
    private static f _borderStyle;

    public static final f getBorderStyle(c cVar) {
        l.f("<this>", cVar);
        f fVar = _borderStyle;
        if (fVar != null) {
            return fVar;
        }
        e eVar = new e("Rounded.BorderStyle", 24.0f, 24.0f, 24.0f, 24.0f, 0L, 0, false, 224);
        int i10 = h0.f9067a;
        m0 m0Var = new m0(t.f6905b);
        n nVarC = gb.e.c(15.0f, 21.0f, 2.0f, -2.0f, -2.0f);
        d.i(nVarC, 2.0f, 19.0f, 21.0f, 2.0f);
        b.h(nVarC, -2.0f, -2.0f, 2.0f);
        a.p(nVarC, 7.0f, 21.0f, 2.0f, -2.0f);
        a.v(nVarC, 7.0f, 19.0f, 2.0f);
        a.p(nVarC, 11.0f, 21.0f, 2.0f, -2.0f);
        d.r(nVarC, -2.0f, 2.0f, 19.0f, 17.0f);
        b.w(nVarC, 2.0f, -2.0f, -2.0f, 2.0f);
        a.p(nVarC, 19.0f, 13.0f, 2.0f, -2.0f);
        d.r(nVarC, -2.0f, 2.0f, 3.0f, 5.0f);
        nVarC.t(15.0f);
        nVarC.i(0.0f, 0.55f, 0.45f, 1.0f, 1.0f, 1.0f);
        nVarC.q(1.0f, -0.45f, 1.0f, -1.0f);
        nVarC.l(5.0f, 6.0f);
        nVarC.i(0.0f, -0.55f, 0.45f, -1.0f, 1.0f, -1.0f);
        nVarC.k(14.0f);
        nVarC.i(0.55f, 0.0f, 1.0f, -0.45f, 1.0f, -1.0f);
        nVarC.q(-0.45f, -1.0f, -1.0f, -1.0f);
        nVarC.l(5.0f, 3.0f);
        nVarC.i(-1.1f, 0.0f, -2.0f, 0.9f, -2.0f, 2.0f);
        a.t(nVarC, 19.0f, 9.0f, 2.0f);
        a.y(nVarC, 21.0f, 7.0f, -2.0f, 2.0f);
        e.a(eVar, nVarC.f3166a, 0, m0Var);
        f fVarB = eVar.b();
        _borderStyle = fVarB;
        return fVarB;
    }
}
